Formula/libpeas.rb

```diff
@@ -10,18 +10,43 @@ class Libpeas < Formula
     sha256 "0f521913ca0eaf13b55aacb75e4b87a730be1f527215741ae2ba207caac523b2" => :el_capitan
   end
 
+  option "with-python", "install support for running Python plugins"
+
   depends_on "gettext" => :build
   depends_on "intltool" => :build
   depends_on "pkg-config" => :build
   depends_on "glib"
   depends_on "gobject-introspection"
   depends_on "gtk+3"
+  depends_on :python => "with-python"
+  depends_on :python3 => "with-python"
+  depends_on "pygobject3" if build.with? "python"
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}",
-                          "--enable-gtk"
+    args = %W[
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
+      --enable-gtk
+    ]
+
+    # Unset PYTHONPATH as it creates more confusion that it tries to
+    # remove when both v2 and v3 are installed
+    ENV.delete("PYTHONPATH")
```

Comment:
I think this isn't set anymore, is it?

Reply:
IIRC as long as Homebrew's `python` is used rather than the system's.

Reply:
I mean … there is no python req anymore so … where would it be set?

Reply:
_Potentially_ when building from source, there's a `with/without-python` option and Homebrew's `python` isn't installed, if I'm reading [this blob](https://github.com/Homebrew/brew/blob/5496a04dd977b1e0d6c71ed0ff6de01e722ca513/Library/Homebrew/language/python.rb#L16-L29) correctly. But, 🤷‍♂️.

Reply:
I get what you're saying though. Practically speaking it should never be set by default any more.

Reply:
Ug. That's depressing

Reply:
The unsetting was certainly needed on my system (else I wouldn't have put it there in the first place).

If it's no longer set on "modern" installs, then the unsetting wouldn't harm either, while still allowing it wo work on "classic" installs?

Reply:
@c-alpha there's no such thing as "modern" vs. "classic" as we only support the HEAD of homebrew/core.

Reply:
Sorry for having been unclear. I was referring to the python install. If you are using the version that ships with macOS or still have a manually installed one somewhere from yesteryear ("classic"), then you might need the `ENV.unset` bit. If you have python installed from homebrew ("modern"), then you may not need it.

So in summary, it would seem that by dropping the `ENV.unset` line, we could potentially break the install for a certain clientele?

Reply:
we filter out all your environment variables